vspace/pom.xml

```diff
@@ -25,8 +25,9 @@
         <db.url>jdbc:mysql://localhost:3306/vspace?serverTimezone=UTC&amp;useEncoding=true</db.url>
         <db.user>vspace</db.user>
         <db.password>vspace</db.password>
-        <uploaded.files.path></uploaded.files.path>
-
+        <uploaded.files.path>${project.basedir}/uploads</uploaded.files.path>
+        <downloaded.files.path>${project.basedir}/downloads</downloaded.files.path>
+        <resources.path>${project.basedir}/src/main/webapp/resources</resources.path>
```

Comment:
I believe `project.basedir` is an absolute path to the base directory of the project? This does not work, as the war will be build on a different system than where it is deployed, so the directory will be a different one. where ever those variables (`uploaded.files.path`, etc) are being used, it should use relative paths for directories that are within the project, and absolute paths set through configurations for folders outside of the project..